parser/src/parser/context.rs

```diff
@@ -60,6 +66,16 @@ impl ParserContext {
         }
     }
 
+    /// Returns the current token if there is one.
+    pub fn curr(&self) -> Option<&SpannedToken> {
```

Comment:
I see that this method is similar to `bump` but returns a reference.
Can we unify the naming to `bump` + `bump_ref` ?
Or `next` and `next_ref`?
Or `last` and `pop` (possibly breaking conventional parser naming but mirroring the rust slice methods)

Reply:
Bump is for progressing a token. Curr is to check if the current token exists and return an option. This has been renamed peek_option.